tasks/enable.yml

```
@@ -0,0 +1,100 @@
+---
+- name: Check trust compatibility
+  fail:
+    msg: Fapolicyd does not support trust setting fapolicyd_setup_trust
+  ignore_errors: true
+  when:
+    - fapolicyd_setup_trust | length > 0
+    - ansible_facts.distribution_version is version("8.2", "<=")
+  register: __failed_check_trust
+
+- name: Check integrity compatibility
+  fail:
+    msg: Fapolicyd does not support integrity setting fapolicyd_setup_integrity
+  ignore_errors: true
+  when:
+    - fapolicyd_setup_integrity | length > 0
+    - ansible_facts.distribution_version is version("8.3", "<=")
+  register: __failed_check_integrity
+
+- name: Check trust files compatibility
+  fail:
+    msg: Fapolicyd does not support trust files setting fapolicyd_add_trusted_file
+  ignore_errors: true
+  when:
+    - fapolicyd_add_trusted_file | length > 0
+    - ansible_facts.distribution_version is version("8.3", "<=")
+  register: __failed_check_trusted_file
+
+- name: Check failed conditions
+  fail:
+    msg: Multiple failed conditions
+  #failed_when: true
+  when: __failed_check_trust is failed or __failed_check_integrity is failed  or
+  __failed_check_trusted_file is failed
+
+- name: Install fapolicyd packages
+  package:
+    name:
+      - fapolicyd
+    state: present
+
+- name: Install fapolicyd-selinux packages
+  package:
+    name:
+      - fapolicyd-selinux
+    state: present
+  when: ansible_facts.distribution_version is version("8.3", ">=")
+
+- name: Copy fapolicyd configuration file
+  template:
+    src: "{{ __fapolicyd_conf }}.j2"
+    dest: "{{ __fapolicyd_dir }}/{{ __fapolicyd_conf }}"
+    owner: root
+    group: fapolicyd
+    mode: '0644'
+
+- name: Run fapolicyd configuration check
+  command: fapolicyd-cli --check-config
+  check_mode: false
+  changed_when: false
+  when: ansible_facts.distribution_version is version("8.6", ">=")
+
+- name: Trustdb cleanup
+  command: fapolicyd-cli --file delete /
+  when: fapolicyd_add_trusted_file | length > 0
```

```suggestion
  when: fapolicyd_add_trusted_file | length > 0
  changed_when: true
```
if the command is executed, it will change the state of the system